{"total":1,"p":1,"ps":10,"rules":[{"key":"csharpsquid:S108","repo":"csharpsquid","name":"Nested blocks of code should not be left empty","createdAt":"2014-06-02T21:50:18+0300","severity":"MAJOR","status":"READY","internalKey":"S108","isTemplate":false,"tags":[],"sysTags":[],"lang":"cs","langName":"C#","htmlDesc":"<p>\nMost of the time a block of code is empty when a piece of code is really missing. So such empty block must be either filled or removed.\n</p>\n\n<h2>Noncompliant Code Example</h2>\n<pre>\nfor (int i = 0; i < 42; i++){}  // Empty on purpose or missing piece of code ?\n</pre>","defaultDebtChar":"RELIABILITY","defaultDebtSubChar":"LOGIC_RELIABILITY","debtChar":"RELIABILITY","debtSubChar":"LOGIC_RELIABILITY","debtCharName":"Reliability","debtSubCharName":"Logic","defaultDebtRemFnType":"LINEAR","defaultDebtRemFnCoeff":"5min","debtOverloaded":false,"debtRemFnType":"LINEAR","debtRemFnCoeff":"5min","params":[]}],"actives":{"csharpsquid:S108":[{"qProfile":"cs-sonar-way-44960","inherit":"NONE","severity":"MAJOR","params":[]}]}}
